packages/template-retail-react-app/app/pages/account/index.jsx

```diff
@@ -91,10 +93,7 @@ const Account = () => {
     // Using Redirect allows us to store the directed page to location
     // so we can direct users back after they are successfully log in
     if (customer.authType != null && !customer.isRegistered) {
-        const path = buildPathWithUrlConfig('/login', {
-            locale: locale.alias || locale.id,
-            site: site.alias || site.id
-        })
+        const path = fillUrlTemplate('/login', site.alias || site.id, locale)
```

Comment:
Should we also parse the locale like this?
```suggestion
        const path = fillUrlTemplate('/login', site.alias || site.id, locale.alias || locale.id)
```

Reply:
We don't have the concept of `locale.alias` in our App configuration. 

Here we're obtaining the locale using the useLocale hook that returns the locale reference. https://github.com/SalesforceCommerceCloud/pwa-kit/blob/b756dab83445b79d44ece98dabf2afe05f13e826/packages/template-retail-react-app/app/pages/account/index.jsx#L48